main.go

```diff
@@ -373,7 +375,7 @@ func runWebhookServer(cmd *cobra.Command, _ []string) {
 func setup(cmd *cobra.Command, log logOptions, telemetry telemetryOptions) {
 	setupLogger(log)
 
-	logger.Info("booting up authorino", "version", version, "cmd", cmd.Use)
+	logger.Info("", "version", version, "commit", gitSHA, "dirty", dirty)
```

Comment:
I would not leave that empty. `authorino` is not needed, but something else?

Reply:
`cmd` must stay too.

Reply:
I can add `"build information"` ?

Reply:
> I can add `"build information"` ?

If you're referring to "cmd", it's not that. cmd is if the binary is being execute as `server`, or `webhooks`, etc

Reply:
@guicassolato let me know if this is what you were looking for.

Reply:
@guicassolato just checking in if the new changes satisfy your comments ?